server/channels/app/plugin.go

```diff
@@ -1266,3 +1267,118 @@ func (ch *Channels) IsPluginActive(pluginName string) (bool, error) {
 
 	return pluginStatus.State == model.PluginStateRunning, nil
 }
+
+// CallPluginBridge executes a bridge call from one plugin to another, or from core to a plugin
+// via HTTP. This enables RESTful bidirectional communication beyond the standard plugin API.
+//
+// Parameters:
+//   - rctx: Request context for logging and tracing
+//   - sourcePluginID: ID of the calling plugin (empty string if called from core)
+//   - targetPluginID: ID of the plugin to call
+//   - endpoint: REST endpoint path to call on the target plugin (e.g., "/api/v1/completion")
+//   - requestData: JSON-encoded request body
+//   - responseSchema: JSON schema or example structure defining expected response format (can be nil)
+//
+// Returns the JSON-encoded response from the target plugin, or an error.
+func (a *App) CallPluginBridge(rctx request.CTX, sourcePluginID, targetPluginID, endpoint string, requestData []byte, responseSchema []byte) ([]byte, error) {
+	pluginsEnvironment := a.GetPluginsEnvironment()
+	if pluginsEnvironment == nil {
+		return nil, errors.New("plugins are not initialized")
+	}
+
+	// Verify target plugin exists and is active
+	if !pluginsEnvironment.IsActive(targetPluginID) {
+		return nil, errors.Errorf("target plugin is not active: %s", targetPluginID)
+	}
+
+	// Construct the HTTP request
+	// PluginHTTP expects path format: /<targetPluginID>/<endpoint>
+	requestURL := fmt.Sprintf("/%s%s", targetPluginID, endpoint)
+	
+	httpReq, err := http.NewRequest(http.MethodPost, requestURL, bytes.NewReader(requestData))
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to create HTTP request")
+	}
+
+	// Set headers
+	httpReq.Header.Set("Content-Type", "application/json")
+	httpReq.Header.Set("X-Mattermost-Request-Id", model.NewId())
+	httpReq.Header.Set("User-Agent", "Mattermost-Plugin-Bridge/1.0")
```

Comment:
`ServeInterPluginRequest` already sets `User-Agent` and request ID in the plugin context